### DISCUSSION TITLE
Add "LEAVE PAYMENT REASON BLANK" to payment details panel

### DISCUSSION THREAD

**Discussion by w0000000t** [bisq-network/bisq#6212] [Ideas]
As per [this forum thread](https://bisq.community/t/why-is-bisq-not-more-clear-about-the-requirement-to-not-enter-anything-as-a-reason-for-payment/11545), I think it would be useful to insert a clear warning to the user about NOT placing anything into the payment reason field of their bank/service, be them the buyer or the seller (sometimes sellers will have the opportunity to add a note to the received payment, for example with interac, and it's just as bad if they place "bitcoin", "bisq" or tradeid in there after buyer pays, as it's happened sometimes in the recent past, already).
Also a `(i)` icon near the warning would expand to say "if you must insert a payment reason as a requirement, either place your own name/username in there, or a dash"

**Comment by ghost** [comment]
We do already provide a clear warning, it reads as follows:


    Important: when making the payment, leave the "reason for payment" field 
    empty. DO NOT put the trade ID or any other text like 'bitcoin', 'BTC', or 'Bisq'. 
    You are free to discuss via trader chat if an alternate "reason for payment" would be suitable to you both.


<details><summary>Zelle (USD)</summary>

![image](https://user-images.githubusercontent.com/47253594/169303549-234c8930-28bb-4d24-b6a6-6f7ca758e2be.png)

</details>


<details><summary>Interac (CAD)</summary>

![image](https://user-images.githubusercontent.com/47253594/169304255-49118c59-eff4-4206-9bad-d88d2f6f578a.png)

</details>


<details><summary>SEPA (EUR)</summary>

![image](https://user-images.githubusercontent.com/47253594/169305011-3b4a29c0-9c75-4bdc-a04d-145f6c044312.png)

</details>


<details><summary>Faster Payments (GBP)</summary>

![image](https://user-images.githubusercontent.com/47253594/169305603-4c7de4b1-444f-4604-b365-f119c719aaf7.png)

</details>

**Comment by w0000000t** [comment]
oh okay! Sorry for bringing this up here instead of having shown this to the user on the forum 😅 
I would like to add though, and consider this as a random flying consideration, that placing the same warning together with the data grid to show the payment details after the trade has been opened (e.g. where you would see beneficiary name, IBAN, BIC, bank name, fiat amount), would increase the reception by the buyers, as it will stay there, always visible, even after the warning popup has been closed, and I believe many (I'm afraid to admit I am one of those!) will close it even before actually having read it all.
Instead, while they are copying each field of the grid in their home banking form, they are bound to hit the "leave reason blank" warning yet once more

**Reply by ghost** [reply]
It's a very good point - moving the reason for payment warning to the grid as you propose would seem to be more effective.  If the community is "for" that change I'll implement it.  @pazza83  :eyes:

**Reply by w0000000t** [reply]
Maybe non even moving that warning, but extending it... the notification popup will stay, so you could leave the "no reason" warning in there, and put that also in the data grid, so it becomes part of the flow. The more you remind users, the better!

**Reply by pazza83** [reply]
The more places it is in the better :)
